skatepark/sass/block-patterns/_mixed-media-in-container.scss

```diff
@@ -0,0 +1,6 @@
+.skatepark-mixed-media-in-container {
+	hr, p, h1, h2, h3, h4, h5, h6 { // Shrink default vertical margins for all the elements in this pattern
```

Comment:
I'm not crazy about this but not sure of a better way to reduce and standardize the margins between headings, paragraphs, and separators for this pattern.

Reply:
I can't think of another way to target these elements... I think this is fine, especially as it's specific to this pattern.

Reply:
makes sense to me that we do it this way, since we are adding the margin globally from the theme css, it's not really the editor that's failing us here. In the end this is just an exception, right?

Reply:
Yeah they are exceptions. But if we [reduce the vertical margins](https://github.com/Automattic/themes/pull/4444#issuecomment-902833612), I don't think these exceptions will be needed.